README.md

```diff
@@ -4,9 +4,113 @@
 
 The Helm Broker is a [Service Broker](https://kyma-project.io/docs/master/components/service-catalog/#service-brokers-overview) which exposes Helm charts as Service Classes in the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-overview). To do so, the Helm Broker uses the concept of addons. An addon is an abstraction layer over a Helm chart which provides all information required to convert the chart into a Service Class.
 
-For more information, read the [Helm Broker documentation](https://kyma-project.io/docs/master/components/helm-broker/).
+For more information, read the [Helm Broker documentation](https://kyma-project.io/docs/master/components/helm-broker/). If you need Helm-Broker installed with all dependencies try [Kyma](https://kyma-project.io/).
 
-## Prerequisites
+## Installation 
+
+To run Helm-Broker you need a Kubernetes cluster with Tiller and Service-Catalog installed. To setup Helm-Broker in Minikube with all necessary dependencies follow these steps:
+If you don't have Minikube installed, follow the [document](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+
+Run the Minikube:
+```bash
+minikube start
+``` 
+
+If you don't have helm installed already, download [the Helm CLI](https://github.com/kubernetes/helm#install).
+Install Tiller into your cluster:
+```bash
+helm init
+```
+
+Install Service-Catalog as a Helm chart:
+```bash
+helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
+helm install svc-cat/catalog --name catalog --namespace catalog
+```
+
+Install Helm-Broker chart:
+```bash
+helm install charts/helm-broker --name helm-broker --namespace helm-broker
+```
+## Usage
+
+This section explains how to use the Helm-Broker.
+
+### Provisioning services provided by the Helm-Broker
+
+If you have installed Helm-Broker with Service-Catalog, you can add an Addon repository and provision Redis instance.
```

Comment:
Why do we use Redis example here? Can't we make it more generic?

Reply:
This is an example of usage. I wanted to show a real case.